prereise/cli/helpers.py

```diff
@@ -0,0 +1,46 @@
+import os
+from datetime import datetime
+
+
+def validate_date(date_string):
+    """Helper function to validate date strings
+
+    :param str date_string: date in string format to be validated
+    :return: (*str*) validated date string
+    """
+    datetime.strptime(date_string, "%Y-%m-%d")
+    return date_string
+
+
+def validate_file_path(file_path):
+    """Helper function to validate file paths
+
+    :param str file_path: path include filename for where
+        to save the file
+    :raises Exception: exception telling user to pick a
+        valid file path
+    :return: (*str*) validated file path
+    """
+    if not os.access(os.path.dirname(file_path), os.W_OK):
+        raise Exception("Please choose a valid file path")
```

Comment:
I would use `ValueError` here, and for the docstring suggest something like `:raises ValueError: if the provided file path is not accessible`

Reply:
Thanks! Fixed